dali/python/nvidia/dali/ops.py

```diff
@@ -449,6 +451,11 @@ def __call__(self, *inputs, **kwargs):
         return outputs
 
 
+def current_dali_stream():
```

Comment:
With this, now a user can do:
```
import nvidia.dali.ops as ops
ops.current_dali_stream()
```
and that will give the impression that this is some general CUDA stream related to the whole DALI but it is only for PythonFunction, isn't it?

Reply:
I made this function a static member of `ops.PythonFunction`